{"tests/test_sync.py::TestSync::test_do_sync": {"type": "victim", "detected_sequence": [["tests/test_sync.py::TestSync::test_set_source_abs_device_relative_path", "tests/test_sync.py::TestSync::test_set_source_abs_not_directory", "tests/test_sync.py::TestSync::test_get_sync_data", "tests/test_sync.py::TestSync::test_do_sync_overwrite", "tests/test_sync.py::TestSync::test_verbose_active", "tests/test_sync.py::TestSync::test_set_destination_abs_absolute_path", "tests/test_sync.py::TestSync::test_gvfs_wrapper_common_exception", "tests/test_sync.py::TestSync::test_verbose_inactive", "tests/test_sync.py::TestReadLink::test_readlink_tilde", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_no_files", "tests/test_sync.py::TestSync::test_sync_handle_unmatched", "tests/test_sync.py::TestSync::test_sync_no_data", "tests/test_sync.py::TestSync::test_get_destination_subdir_data_", "tests/test_sync.py::TestReadLink::test_readlink_device_path", "tests/test_sync.py::TestSync::test_set_destination_abs_device_relative_path", "tests/test_sync.py::TestSync::test_set_source_abs_computer_relative_path", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_remove", "tests/test_sync.py::TestSync::test_get_destination_subdir_data_doesnt_exist", "tests/test_sync.py::TestSync::test_set_source_abs_absolute_path", "tests/test_sync.py::TestSync::test_sync_ignore_unmatched", "tests/test_sync.py::TestSync::test_gvfs_wrapper_bash_exception_any", "tests/test_sync.py::TestSync::test_set_source_abs_nonexistent", "tests/test_sync.py::TestSync::test_sync_data_template", "tests/test_sync.py::TestSync::test_get_source_subdir_data", "tests/test_sync.py::TestReadLink::test_readlink_slash", "tests/test_sync.py::TestReadLink::test_readlink_no_path", "tests/test_sync.py::TestSync::test_do_sync"], ["tests/test_find_device.py::TestLsusb::test_lsusb", "tests/test_find_device.py::TestFindDevice::test_get_mtp_details", "tests/test_find_device.py::TestFindDevice::test_get_connection_details_device_exception_message", "tests/test_find_device.py::TestFindDevice::test_get_connection_details_multiple_devices", "tests/test_find_device.py::TestFindDevice::test_get_connection_details_device_exception", "tests/test_gvfs.py::TestGvfsWrappers::test_mkdir", "tests/test_gvfs.py::TestGvfsWrappers::test_mv", "tests/test_gvfs.py::TestGvfsWrappers::test_mount", "tests/test_gvfs.py::TestGvfsWrappers::test_rm", "tests/test_gvfs.py::TestGvfsWrappers::test_cp", "tests/test_cli.py::TestCli::test_main", "tests/test_cli.py::TestCli::test_sync_info_file_incorrect_format", "tests/test_cli.py::TestCli::test_parser_unmatched", "tests/test_cli.py::TestCli::test_sync_info_args", "tests/test_cli.py::TestCli::test_parser", "tests/test_cli.py::TestCli::test_parser_ignored", "tests/test_cli.py::TestCli::test_run", "tests/test_cli.py::TestCli::test_run_sync_info_exception", "tests/test_cli.py::TestCli::test_sync_info_file", "tests/test_cli.py::TestCli::test_sync_info_args_and_file_are_exclusive", "tests/test_cli.py::TestCli::test_sync_info_missing", "tests/test_cli.py::TestCli::test_parser_no_args", "tests/test_cli.py::TestCli::test_run_device_exception", "tests/test_sync.py::TestSync::test_copy_file", "tests/test_sync.py::TestSync::test_verbose_active", "tests/test_sync.py::TestSync::test_verbose_inactive", "tests/test_sync.py::TestSync::test_sync_ignore_unmatched", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_sync", "tests/test_sync.py::TestReadLink::test_readlink_tilde", "tests/test_sync.py::TestSync::test_sync_handle_unmatched", "tests/test_sync.py::TestSync::test_set_source_abs_nonexistent", "tests/test_sync.py::TestSync::test_get_destination_subdir_data_doesnt_exist", "tests/test_sync.py::TestSync::test_get_source_subdir_data", "tests/test_sync.py::TestReadLink::test_readlink_slash", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_remove", "tests/test_sync.py::TestSync::test_set_source_abs_computer_relative_path", "tests/test_sync.py::TestSync::test_gvfs_wrapper_bash_exception_any", "tests/test_sync.py::TestSync::test_gvfs_wrapper_bash_exception_exact", "tests/test_sync.py::TestSync::test_sync_no_data", "tests/test_sync.py::TestSync::test_set_destination_abs_device_relative_path", "tests/test_sync.py::TestReadLink::test_readlink_nonexisting", "tests/test_sync.py::TestSync::test_set_source_abs_device_relative_path", "tests/test_sync.py::TestSync::test_handle_ignored_file_type", "tests/test_sync.py::TestSync::test_gvfs_wrapper_common_exception", "tests/test_sync.py::TestSync::test_set_destination_abs_absolute_path", "tests/test_sync.py::TestSync::test_set_destination_abs_computer_relative_path", "tests/test_sync.py::TestSync::test_set_destination_subdir_absh", "tests/test_sync.py::TestSync::test_do_sync_overwrite", "tests/test_sync.py::TestReadLink::test_readlink_device_path", "tests/test_sync.py::TestSync::test_set_source_abs_not_directory", "tests/test_sync.py::TestSync::test_do_sync"], ["tests/test_sync_integration.py::TestSyncIntegration::test_sync_removes_unmatched", "tests/test_sync_integration.py::TestSyncIntegration::test_sync_default", "tests/test_sync_integration.py::TestSyncIntegration::test_sync_synchronizes_unmatched", "tests/test_cli.py::TestCli::test_run_device_exception", "tests/test_cli.py::TestCli::test_parser", "tests/test_cli.py::TestCli::test_parser_ignored", "tests/test_cli.py::TestCli::test_parser_unmatched", "tests/test_cli.py::TestCli::test_main", "tests/test_cli.py::TestCli::test_run", "tests/test_cli.py::TestCli::test_sync_info_file", "tests/test_cli.py::TestCli::test_run_sync_info_exception", "tests/test_cli.py::TestCli::test_sync_info_args_and_file_are_exclusive", "tests/test_cli.py::TestCli::test_parser_no_args", "tests/test_cli.py::TestCli::test_sync_info_file_incorrect_format", "tests/test_cli.py::TestCli::test_sync_info_args", "tests/test_cli.py::TestCli::test_sync_info_missing", "tests/test_sync.py::TestSync::test_set_destination_abs_computer_relative_path", "tests/test_sync.py::TestSync::test_set_destination_subdir_absh", "tests/test_sync.py::TestSync::test_get_sync_data", "tests/test_sync.py::TestSync::test_sync_handle_unmatched", "tests/test_sync.py::TestSync::test_verbose_inactive", "tests/test_sync.py::TestSync::test_do_sync_overwrite", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_sync", "tests/test_sync.py::TestReadLink::test_readlink_device_path", "tests/test_sync.py::TestSync::test_set_source_abs_device_relative_path", "tests/test_sync.py::TestSync::test_set_source_abs_not_directory", "tests/test_sync.py::TestReadLink::test_readlink_slash", "tests/test_sync.py::TestSync::test_get_source_subdir_data", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_remove", "tests/test_sync.py::TestSync::test_get_destination_subdir_data_", "tests/test_sync.py::TestSync::test_handle_destination_dir_data_no_files", "tests/test_sync.py::TestSync::test_set_destination_abs_absolute_path", "tests/test_sync.py::TestSync::test_set_source_abs_absolute_path", "tests/test_sync.py::TestSync::test_set_source_abs_computer_relative_path", "tests/test_sync.py::TestSync::test_gvfs_wrapper_bash_exception_any", "tests/test_sync.py::TestSync::test_gvfs_wrapper_common_exception", "tests/test_sync.py::TestSync::test_copy_file", "tests/test_sync.py::TestSync::test_set_source_abs_nonexistent", "tests/test_sync.py::TestSync::test_get_destination_subdir_data_doesnt_exist", "tests/test_sync.py::TestReadLink::test_readlink_nonexisting", "tests/test_sync.py::TestSync::test_set_destination_abs_device_relative_path", "tests/test_sync.py::TestReadLink::test_readlink_no_path", "tests/test_sync.py::TestReadLink::test_readlink_tilde", "tests/test_sync.py::TestSync::test_do_sync"]]}, "time": 88.15299153327942}